packages/solo/src/web.js

```diff
@@ -194,6 +198,33 @@ export async function makeHTTPListener(
       .catch(_ => {});
   });
 
+  app.get('/connections', async (req, res) => {
+    if (!(await validateAccessToken(req))) {
+      res.json({ ok: false, rej: 'Unauthorized' });
+      return;
+    }
+
+    res.json({
+      ok: true,
+      connections,
+    });
+  });
+
+  app.post('/publish-bundle', async (req, res) => {
+    if (!(await validateAccessToken(req))) {
+      res.json({ ok: false, rej: 'Unauthorized' });
+      return;
+    }
+    try {
+      const bundle = harden(req.body);
+      await validateAndInstallBundle(bundle);
```

Comment:
I'm not sure how it should work / already works in the solo machine, but after doing `validateAndInstallBundle`, the host app is supposed to commit the DB changes. Did you run into anything that might do a swingstore commit?